x-pack/filebeat/module/aws/cloudtrail/manifest.yml

```diff
@@ -13,6 +13,8 @@ var:
   - name: secret_access_key
   - name: session_token
   - name: role_arn
+  - name: tags
+    default: [cloud]
```

Comment:
also, same here, i'm thinking we should probably just tag all the resources for `cloud` with `forwarded` too, no?